golang-providers/pe-golang-bin/deb/build.sh

```diff
@@ -3,7 +3,7 @@
 # Internal variables
 PELION_PACKAGE_NAME="pe-golang-bin"
 PELION_PACKAGE_DIR=$(cd "`dirname \"$0\"`" && pwd)
-PELION_PACKAGE_TARBALL=go1.16.15.linux-amd64.tar.gz
+PELION_PACKAGE_TARBALL=go1.19.13.linux-amd64.tar.gz
```

Comment:
pe-golang-bin and golang-virtual are mutually exclusive.
pe-golang-bin is used when building for ubuntu/bionic, debian/buster and debian/bullseye. All other environments use golang-virtual.
Therefore for the newer platforms we will use golang 1.20 and for the 3 mentioned above we will use 1.19.
This could be revisited in the future and pe-golang-bin could be fixed to use 1.20.

Reply:
I've updated the README.md in golang-providers to link to where the versions are specified.

Reply:
Great!